test/unit/spec/signaling/v2/room.js

```diff
@@ -1901,7 +1919,12 @@ function makePeerConnectionManager(getRoom) {
 
     return new Map([
       ['foo', {
-        activeIceCandidatePair: { baz: 'zee' },
+        activeIceCandidatePair: {
+          baz: 'zee',
+          localCandidate: {
+            relayProtocol: 'bar'
```

Comment:
I think we either need to parameterize this or we need to use meaningful names so that we know one of the peerConnection has relayProtocol set and other one not.